### ISSUE TITLE
Incorrect test data

### ISSUE THREAD

**Issue by realindiahotel** [bip32JP/bip32JP.github.io#4]
Hello, @bip32JP you don't seem to respond to my comments on the closed pull request.

The test data you have listed is wrong. As we are building unit tests to test against the test data you supply, it is easy to see that your test data is invalid because it contains ideographic spaces.

As you can see in the BIP39 spec, the mnemonic sentence generated will always contain words separated by an ASCII space. I realise you are saying that people need to normalise the test data before using it but this is incorrect practice. Test data should ALWAYS be an exact match of what you expect the code to produce. If you leave that test data there with ideographic spaces and someone fails to normalise the test data and gets the code working to your test data that would be very bad.

Long story short, test data to be correct to the BIP39 spec can NEVER contain ideographic spaces as the code normalises all output. Therefore correct code will always fail your test data.

If you want to refute my claim, can you please show me the code in the reference client that outputs a mnemonic with ideographic spaces? Else please explain how it is possible for the reference code to produce data that matches your test output.
